MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Selections in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about Selections feature in Syncfusion .NET MAUI Calendar (SfCalendar) control and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Selections in .NET MAUI Calendar (SfCalendar)
+The Selection mode is specified in the Calendar property enumeration. You can select the dates by tapping the cell in the calendar. `SfCalendar` provides three types of modes such as `Single`, `Multiple`, and `Range` Selection. The default [SelectionMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_SelectionMode) is Single, which allows the user to select only one date at a time.
+
+>**NOTE** You can select the cells in the Year, Decade, and Century views in the calendar only when the `AllowViewNavigation` property is set to false.
+
+## Single selection
+The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single`. In this selection, you can select a single date from the cell.
```

Comment:
you can select a single date from the cell -- Is that express the right meaning

Reply:
Content Modified